pkg/scanners/trivy/trivy.go

```diff
@@ -271,5 +275,19 @@ func main() {
 		log.Info("garbage in pipe", "pipeName", util.EraseCompleteScanPath, "in_pipe", string(data))
 		os.Exit(1)
 	}
+
+	// record  metrics
+	ctx, cancel := signal.NotifyContext(context.Background(), os.Interrupt, syscall.SIGTERM)
+	defer cancel()
+
+	exporter, reader, provider := metrics.ConfigureMetrics(ctx, log, os.Getenv("OTEL_EXPORTER_OTLP_ENDPOINT"))
```

Comment:
What happens if `os.Getenv("OTEL_EXPORTER_OTLP_ENDPOINT")` is empty?

Reply:
handled that with a check in the most recent commit!